### ISSUE TITLE
D10: Pre-Launch Accessibility testing

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16065]
## Description
In prep for the CMS upgrade to Drupal 10, a final round of pre-launch accessibility testing should be done to make sure that no launch-blocking accessibility issues will be introduced with the upgrade. Any new accessibility issues found, launch-blocking or otherwise should be ticketed to be reviewed and prioritized as needed.

## Acceptance Criteria
- [x] Accessibility review of D10 environment is completed
- [x] Tickets are created for any new issues found
- [x] Product is notified of any launch-blocking issues (if any)

**Comment by BerniXiongA6** [comment]
@laflannery you ARE the absolute BEST! Thank you!
